### ISSUE TITLE
Cannot get keyboard focus on marker to open popup, or trigger a command

### ISSUE THREAD

**Issue by sirleech** [Leaflet/Leaflet#1355]
One cannot get keyboard focus to activate a marker. However, you can access the close button of an already open popup with the keyboard.

Accessibility issue.
